src/Mouf/Html/Widgets/EvoluGrid/SimpleColumn.php

```diff
@@ -153,7 +153,7 @@ public function isSortable() : bool
      */
     public function getSortKey()
     {
-        return $this->key;
+        return $this->sortKey;
```

Comment:
Isn't it the key by default is the sort key is not provided?

Maybe something more like:

```suggestion
        return $this->sortKey ?: $this->key;
```

Reply:
Dont know, in TwigColumn, getter is :
```php
public function getSortKey()
    {
        return $this->sortKey;
    }
```

Reply:
... but I can change it to ```return $this->sortKey ?: $this->key;```

Reply:
Because in TwigColum, there is no "$this->key", right?